adam/curriculum/organize_objects.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import logging
+import shutil
+from pathlib import Path
+from typing import Tuple, Generator
+
+import yaml
+
+from adam.ontology.phase3_ontology import PHASE_3_CURRICULUM_OBJECTS
+
+logger = logging.getLogger(__name__)
+
+
+def img_iter(source_img_dir: Path) -> Generator[Tuple[str, Path], None, None]:
+    """Function to load concept name and image as a single iterable."""
+    for object_concept in PHASE_3_CURRICULUM_OBJECTS:
+        concept_path = source_img_dir / object_concept.handle
+
+        if not concept_path.is_dir():
+            logger.warning(
+                "Concept %s not found in given directory", object_concept.handle
+            )
+            continue
+
+        for img_path in sorted(concept_path.glob("*.png")):
+            yield object_concept.handle, img_path
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(
+        description="Utility script to reorganize Action curricula"
+    )
+    parser.add_argument(
+        "input_img_dir", type=Path, required=True, help="Input set of images."
+    )
+    parser.add_argument(
+        "output_dir", type=Path, required=True, help="Directory to put the curriculum in."
+    )
+    args = parser.parse_args()
+
+    source_img_dir = Path(args.input_img_dir)
+    curriculum_dir = Path(args.output_dir)
+
+    # jal: Should overwrite be optional based on a flag?
```

Comment:
Doesn't matter to me. It's probably easy enough to add this functionality but it's also pretty easy to just manually `rm -r` the output dir.